subprojects/workers/src/main/java/org/gradle/workers/internal/ActionExecutionSpec.java

```diff
@@ -32,11 +34,13 @@
     private final String displayName;
     private final Class<? extends Runnable> implementationClass;
     private final byte[] params;
+    private final OperationIdentifier operationId;
 
-    ActionExecutionSpec(Class<? extends Runnable> implementationClass, String displayName, Object[] params) {
+    ActionExecutionSpec(Class<? extends Runnable> implementationClass, String displayName, Object[] params, OperationIdentifier operationId) {
```

Comment:
This class shouldn't change. It's part of a client of `BuildOperationExecutor` and shouldn't be impacted.